ConsumerApi.Sdk/Endpoints/Files/Types/Requests/CreateFileRequest.cs

```diff
@@ -18,7 +18,7 @@ public MultipartFormDataContent ToMultipartContent()
     {
         return new MultipartFormDataContent
         {
-            { new StreamContent(Content), "content" },
+            { new StreamContent(Content), "content", "test.txt" },
```

Comment:
I know I'm a tester, but anyway... what is this about?

Reply:
I think Nikola said something about that in one of our dailies. He said it was required, or otherwise the request would fail.

Reply:
Precisely, it's not related to tests, it's just that the `HttpContent` requires the file name when passing in the `StreamContent` object as first parameter.